### PR TITLE
feat: Expose vsock::VM_ADDR_* constants

### PR DESCRIPTION
**Pull request by jalil-salame**
These constants are useful when constructing new `VsockAddr`s and are currently not reexported thus making the consuming library/application require to either write the values themselves (very brittle), or add an unnecessary dependency on either nix or vsock.

tokio-vsock already depends on vsock-rs, thus we can just reexport these values.

### CODE REVIEW

**Review by qwandor — APPROVED** [review]

**Comment by qwandor** [comment]
Thanks for the PR! If you edit your commit message to make the presubmit check pass then I'll merge this.

**Comment by jalil-salame** [comment]
It might take a while for me to fix this, between software issues and being on a train it'll take way more time than necessary...

**Comment by qwandor** [comment]
Ah, actually we need a second approver before this can be merged, I forgot about the policy. Hopefully someone else can add their approval soon.

**Review by jeromegn — APPROVED** [review]

**Pull request merged**
